### ISSUE TITLE
Check failures with current RSQLite

### ISSUE THREAD

**Issue by krlmlr** [dbottomly/poplite#2]
I'm getting:

```
* checking tests ...
  Running ‘testthat.R’
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Failure: createTable (@test-poplite.R#252) 
  2. Failure: createTable (@test-poplite.R#252) 
  3. Failure: createTable (@test-poplite.R#252) 
  4. Failure: createTable (@test-poplite.R#252) 
  5. Failure: insertStatement (@test-poplite.R#330) 
  6. Failure: insertStatement (@test-poplite.R#350) 
  7. Failure: insertStatement (@test-poplite.R#330) 
  8. Failure: insertStatement (@test-poplite.R#350) 
  9. Failure: insertStatement (@test-poplite.R#330) 
  1. ...

  Error: testthat unit tests failed
  Execution halted
```

Please let me know if you think that RSQLite needs to be fixed, or update your package for compatibility with current RSQLite. The latest changes are summarized in https://github.com/rstats-db/RSQLite/blob/master/NEWS.md.

Please also consider fixing the [CRAN notes](https://cran.r-project.org/web/checks/check_results_poplite.html).

**Comment by krlmlr** [comment]
These failures are gone after I fixed dbGetException() after dbSendQuery(). Please bear in mind that dbGetException() is deprecated.

**Comment by krlmlr** [comment]
Sorry, I was mistaken -- the failures still occur.

**Comment by dbottomly** [comment]
Errors should be fixed with 0.99.17.3 in devel.
